projects/v2/src/app/components/report/report.service.ts

```diff
@@ -201,8 +201,8 @@ export class ReportService {
         reportdata,
         compareData
       );
-      newEntry.identicalB = identicalStructuresB;
-      newEntry.newB = newStructuresB;
+      newEntry.identicalB = newStructuresB;
```

Comment:
This seems... wrong

Reply:
Ellan wanted identical biomarkers and new biomarkers columns to be reversed. So I have swapped the two variables.

Reply:
Please rename so that it makes sense. This is really sloppy.